Application/EdFi.Ods.Tests/EdFi.Ods.Api/Caching/StubSecurityContext.cs

```diff
@@ -23,21 +20,21 @@ public static ISecurityContext GetMockedSecurityContext()
         {
             var securityContext = A.Fake<ISecurityContext>();
             // The underlying SecurityRepository implementation expects this application, so force it to be there in the fake
```

Comment:
This comment no longer seems relevant. Perhaps it would be good to remove it.

I did not think about the following idea when you were working on the similar class in another project. It might be good to add a comment explaining what this class is doing. Maybe something like this:

```csharp
// This stub class allows us to compile and perform some tests in repositories
// that rely on `ISecurityContext`. However, `DbSet` is very difficult to work with
// in unit tests, and we no longer try. This stub class returns `null` for everything,
// under the assumption that test classes that use this stub are only testing 
// situations where the properties below will not be called.
```

Now, looking at this again, I also realize that we could have gotten rid of `GetFakeDbSet` altogether and instead simply set each of these properties `= null`!